app/views/payments/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render "shared/header", title: 'NEW PAYMENT', icon: 'arrow-back', link: root_path, logout: true %>
```

Comment:
- [ ] When I am on the new transaction page and I click the back arrow I should be taken to the transactions index page, not the root path. Kindly fix that.